packages/api/src/context.ts

```diff
@@ -447,6 +455,14 @@ export async function contextFromRequest(
       if (!updatedPayment) throw new Error('Error during updating payment') // TODO: this check needs to be removed
 
       return updatedPayment
+    },
+
+    async getSystemInformation(): Promise<SystemInformation> {
```

Comment:
A function should always come with a minimum of comments. As a dev I want to know why this function and what it does.